src/main/java/com/example/bankoflykhvar/model/Transaction.java

```diff
@@ -0,0 +1,87 @@
+package com.example.bankoflykhvar.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "transaction")
+public class Transaction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "account_from", nullable = false)
+    private Account accountFrom;
+    @ManyToOne
+    @JoinColumn(name = "account_to", nullable = false)
+    private Account accountTo;
+    private Double amount;
```

Comment:
```suggestion
    private BigDecimal amount;
```
Better use BigDecimal for money fields